gramps/plugins/drawreport/calendarreport.py

```diff
@@ -335,6 +337,16 @@ def collect_data(self):
 
                     prob_alive_date = Date(self.year, month, day)
 
+                    # if birth on february, 29, we must choose
```

Comment:
This code is used 4 times, which indicates that it should be made into a utility function.

Reply:
Do I add this in gramps/gen/lib/date.py ?

Reply:
Yes.  I was thinking something along the lines of:

`day, month = birth_date.anniversary(this_year)`

We can probably code just for the Gregorian calendar to start with.  Other calendars can be added later.